src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/reader/MetricPropertiesConfig.java

```diff
@@ -190,6 +192,8 @@ private MetricPropertiesConfig() {
         MetricName.MOUNTED_PARTITION_METRICS);
     eventKeyToMetricNameMap.put(PerformanceAnalyzerMetrics.sClusterApplierService,
             MetricName.CLUSTER_APPLIER_SERVICE);
```

Comment:
Can we follow the same indentation here.

Reply:
Done. Changed some other entries for match the indentation